src/tests/ftest/util/nvme_utils.py

```diff
@@ -78,48 +80,67 @@ def setUp(self):
         # Start the servers and agents
         super().setUp()
         self.hostfile_clients = None
-        self.ior_default_flags = self.ior_cmd.flags.value
+        self.ior_local_cmd = IorCommand()
+        self.ior_local_cmd.get_params(self)
+        self.ior_default_flags = self.ior_local_cmd.flags.value
         self.ior_scm_xfersize = self.params.get("transfer_size",
                                                 '/run/ior/transfersize_blocksize/*', '2048')
         self.ior_read_flags = self.params.get("read_flags", '/run/ior/iorflags/*', '-r -R -k -G 1')
         self.ior_nvme_xfersize = self.params.get("nvme_transfer_size",
                                                  '/run/ior/transfersize_blocksize/*', '16777216')
         # Get the number of daos_engine
         self.engines = self.server_managers[0].manager.job.yaml.engine_params
-        self.out_queue = queue.Queue()
 
-    def start_ior_thread(self, results, create_cont, operation):
+    def start_ior_thread(self, create_cont, operation):
         """Start IOR write/read threads and wait until all threads are finished.
 
         Args:
-            results (queue): queue for returning thread results
             create_cont (Bool): To create the new container or not.
             operation (str):
                 Write/WriteRead: It will Write or Write/Read base on IOR parameter in yaml file.
                 Auto_Write/Auto_Read: It will calculate the IOR block size based on requested
                                         storage % to be fill.
         """
         # IOR flag can be Write only or Write/Read based on test yaml
-        self.ior_cmd.flags.value = self.ior_default_flags
+        self.ior_local_cmd.flags.value = self.ior_default_flags
 
         # Calculate the block size based on server % to fill up.
         if 'Auto' in operation:
             block_size = self.calculate_ior_block_size()
-            self.ior_cmd.block_size.update('{}'.format(block_size))
+            self.ior_local_cmd.block_size.update('{}'.format(block_size))
 
-        # For IOR Read operation update the read flax from yaml file.
+        # IOR Read operation update the read only flag from yaml file.
         if 'Auto_Read' in operation or operation == "Read":
             create_cont = False
-            self.ior_cmd.flags.value = self.ior_read_flags
+            self.ior_local_cmd.flags.value = self.ior_read_flags
+
+        self.ior_local_cmd.set_daos_params(self.server_group, self.pool)
+        self.ior_local_cmd.test_file.update('/testfile')
+
+        # Created new container
+        if create_cont:
+            self.create_cont()
+        else:
+            self.ior_local_cmd.dfs_cont.update(self.container.uuid)
+
+        # Define the job manager for the IOR command
+        job_manager_main = Mpirun(self.ior_local_cmd, mpitype="mpich")
```

Comment:
(pylint-unexpected-keyword-arg) Unexpected keyword argument 'mpitype' in constructor call